.mdlstyle.rb

```diff
@@ -6,7 +6,7 @@
 exclude_rule 'MD013'
 exclude_rule 'MD024'
 rule 'MD026', :punctuation => ".,;"
-rule 'MD029', :style => :ordered
+rule 'MD029', :style => :one
```

Comment:
Using automatic numbering like this can cause lists with subitems to break. For example

```
1. something

:::{image} path
:alt: Caption
:align: center
:::

1. Something else
```

might show as

```
1. something
1. something else
```

Using `style: ordered` allows you to explicitly set the second bullet to `2.` and avoid this.

Reply:
Tried this locally and the numbering is corrected in the rendered output.
May have been a `myst-parser` bug which is fixed now?

I did see some places where there were separate lists though,

```markdown
1. first list

2. second list
```

which is incorrect.

Reply:
So, I think one is better because the numbers are always correct in the output, we don't have to manually index ordered lists.